### ISSUE TITLE
make command error

### ISSUE THREAD

**Issue by sas05** [deepstreamIO/deepstream.io-benchmarks#3]
os: ubuntu 18.04

I got this following error after run make command.
g++ -std=c++11 eventBenchmark.cpp -luv -lz -lssl -luWS -o eventBenchmark
eventBenchmark.cpp:4:10: fatal error: uWS/uWS.h: No such file or directory
    4 | #include <uWS/uWS.h>
      |          ^~~~~~~~~~~
compilation terminated.
make: *** [Makefile:2: default] Error 1

**Comment by yasserf** [comment]
Yeah this repo is pretty old and has not been updated in a while I’m afraid. It’s also the text protocol which is invalid after 3.0 in terms of server efficiency.
